tgui/packages/tgui/interfaces/Clipboard.js

```diff
@@ -0,0 +1,118 @@
+import { useBackend } from "../backend";
+import {
+  Box,
+  Button,
+  Divider,
+  LabeledList,
+  Flex,
+  Section,
+} from "../components";
```

Comment:
Prettier detected, I dislike that it formats imports like this, even though we have column 80 ignored in eslint specifically for imports. But you can keep it that way, since you intend to continue working on it.

Reply:
Aren't you enforcing an autoformatter onto tgui? See [`9943ddb` (#57289)](https://github.com/tgstation/tgstation/pull/57289/commits/9943ddbbd460e6f8fb676aa5de39d1b269828596), in which I ran `bin/tgui --fix` after trying Prettier (my default) and the default VS Code js / ts lang one

Reply:
No, it's a bit hard to configure eslint for precise autoformatting.

Reply:
The linter was annoyed at styling choices though…? Weird.

Reply:
Only I am annoyed at the moment. 😄